PROJECT_CHARTER.md

```diff
@@ -67,7 +67,35 @@ Section Intentionally Left Blank
 
 ### Section 4.2: Decision-making, Voting, and/or Elections
 
-Section Intentionally Left Blank
+**Decision-making**
+
+The Fastify's features can be discussed in the GitHub's issues and/or projects.
+The consensus on a discussion is reached when:
+
+- the majority of the Collaborator agree on the decision
+- there is not any blocking concern by the Lead Maintainers
+
```

Comment:
I think we can replace the above two points with there is no objection by any of the collaborators. A concept like majority is complex to manage.

Reply:
@Eomm ping ^

We can lift this from https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborators